README.md

```diff
@@ -46,6 +46,11 @@ set CC=clang
 set AR=llvm-ar
 ```
 
+### Secure boot support
+
+See [Secure Boot Guide](secure-boot-guide.md)
```

Comment:
Need move to doc dir.

Also need mention secure_boot.md just after "The API specification is at [td-shim specification]"